docs/content/reloader/monitoring-secrets-configmaps.md

```diff
@@ -1,5 +1,9 @@
 # Monitoring secrets and configMaps for changes with Reloader
-For `deployments` in the Smaug cluster, Reloader can be used to monitor `configMap` and `secret` resources for changes. If a change is detected Reloader will trigger a rolling upgrade on relevant pods via the associated `deployment`.
+For `deployments`, Reloader can be used to monitor `configMap` and `secret` resources for changes. If a change is detected Reloader will trigger a rolling upgrade on relevant pods via the associated `deployment`.
+
+Reloader is currently deployed on the following clusters in Operate First:
```

Comment:
Maybe instead of maintaining a separate list, it might be better just to link this path: 

https://github.com/operate-first/apps/tree/master/reloader/overlays 

So we don't need to update this markdown everytime a new reloader is added.

Reply:
Good call, just updated.